.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!-- Thanks for taking the time to write this Pull Request ❤️ -->
+
+## 🚀 Description
+
+<!-- Describe your changes in detail -->
+
+## 📄 Motivation and Context
+
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## 🧪 How Has This Been Tested?
+
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, tests ran to see how -->
+<!-- your change affects other areas of the code, etc. -->
+
+
+## 📦 Types of changes
+
+<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply and remove the ones not applicable -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## ✅ Checklist
```

Comment:
A checklist might be good, I'm not so sure about the documentation part. I guess nobody will tick any of these. But we can try it out.

Reply:
I removed the checklist for now. We can add it later as it grows :) What say?